pandas/core/arrays/datetimelike.py

```diff
@@ -483,6 +483,26 @@ def _maybe_mask_results(self, result, fill_value=iNaT, convert=None):
             result[self._isnan] = fill_value
         return result
 
+    # ------------------------------------------------------------------
+    # Additional array methods
+    # These are not part of the EA API, but we implement them because
+    # pandas currently assumes they're there.
+
+    def view(self, dtype=None):
```

Comment:
are we adding to EA interface? or just here (ok either way). @TomAugspurger ?

Reply:
I don't think ExtensionArray.view makes sense. The result of that would (I think) depend on the memory layout of the concrete array backing the ExtesnionArray, and I think that ExtensionArray should remain agnostic to that.